dns/zone-configs/k8s.io._0_base.yaml

```diff
@@ -326,16 +326,10 @@ submit-queue:
 testgrid:
   - type: A
     values:
-    - 216.239.32.21
-    - 216.239.34.21
-    - 216.239.36.21
-    - 216.239.38.21
+    - 34.120.51.46
   - type: AAAA
     values:
-    - 2001:4860:4802:32::15
-    - 2001:4860:4802:34::15
-    - 2001:4860:4802:36::15
-    - 2001:4860:4802:38::15
+    - 2600:1901:0:dc01::
```

Comment:
this needs to be quoted, yaml parses this like `foo:` (a key `2600:1901:0:dc01:` with key/value delimeter `:` and value `nulll`)

Reply:
Ah yup, thanks for the tip!